spec/system/submissions/review_spec.rb

```diff
@@ -720,6 +724,13 @@
       expect(page).to have_text(submission_reviewed_3.checklist.first['title'])
       expect(page).to have_text(submission_reviewed_2.checklist.first['title'])
       expect(page).not_to have_text(submission_reviewed_1.checklist.first['title'])
+
+      # submission 4
+      visit review_timeline_event_path(submission_reviewed_4)
+
+      expect(page).to have_text(team_1.founders.last.name)
+      expect(page).to have_text(team_1.founders.first.name)
+      expect(page).to have_text(team_1.name)
```

Comment:
Because this is a modification to an existing spec, I went through it, and found it difficult to figure out what it's testing. The main issue is what the spec _says_ it's does:

```
context: with reviewed submissions from 2 teams
  scenario: coach visits submission review page
```

The problem is that it doesn't even _attempt_ to describe what is going to be tested.

What this spec is actually testing is the selection of which _other_ submissions to show when a particular submission's overlay is opened. We have special logic which dictates that only those other submissions for the same target which have a matching list of submission owners should be displayed in the overlay (as other related submissions).

Given that this is the case, this context / scenario is **definitely** not the place to be adding a test for this current change - which is very much unrelated. It should be in a separate scenario.

Also, since the poor context and scenario naming has led to the addition of the spec here, I think this particular spec should also be renamed and documented better so as to avoid confusion.

```
context: when there are some submissions that have a mixed list of owners
  scenario: coach viewing a submission's review page is only shown other submissions with identical owners
```